src/TextLineSpacingCommand.js

```diff
@@ -85,12 +84,11 @@ export function setTextLineSpacing(
 }
 
 function createGroup(): Array<{[string]: TextLineSpacingCommand}> {
-  const group = {};
-  group['Single'] = new TextLineSpacingCommand(null);
-  LINE_SPACING_VALUES.forEach(lineSpacing => {
-    const label = lineSpacing === '200%' ? 'Double' : ` ${lineSpacing} `;
-    group[label] = new TextLineSpacingCommand(lineSpacing);
-  });
+  const group = {
+    Single: new TextLineSpacingCommand(SINGLE_LINE_SPACING),
+    ' 150% ': new TextLineSpacingCommand('150%'),
```

Comment:
what are the extra spaces around the 150% for?

Reply:
That's for the label text.